### ISSUE TITLE
Recipe Page and Single Cookbook Back Buttons

### ISSUE THREAD

**Issue by grantDuntugan** [cse110-fa21-group24/cse110-fa21-group24#171]
**Description**
1. Add back buttons to the recipe pages and when viewing a single cookbook.
2. When you click on the back buttons, you should be able to navigate to the page that the user was previously on (this will either be the Home page, Explore page, Single Cookbook page when on the recipe page and the My Cookbooks page when in the Single Cookbook view).

**Possible Solutions**
- You will most likely need to restyle the recipe pages as well as mess around with the `Router` class a bit. Make sure all style changes are responsive to the size of the viewport.
- Split this task into parts as you see fit and work together on this!

**Issue Points**: 2
